packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -111,6 +111,11 @@
 	.dataviews-pagination {
 		z-index: z-index(".edit-site-patterns__dataviews-list-pagination");
 	}
+
+	.dataviews-view-grid__field-sync-status.is-badge:has(.edit-site-patterns__field-sync-status-fully) {
```

Comment:
I'd prefer if we don't target dataviews classes in consumer code, as it implies it's a public API. Can we instead add a specific class in the `sync-status` field? We have access to the view type there.

I've noticed that we do this for [pagination](https://github.com/WordPress/gutenberg/pull/58965/files#diff-51f269c9a5ba27a60e9fa5b9ca2b38716feb71521443e44c542f2e0423ee3dec), [pattern actions](https://github.com/WordPress/gutenberg/pull/57592/files#diff-51f269c9a5ba27a60e9fa5b9ca2b38716feb71521443e44c542f2e0423ee3dec), and [templates/parts actions](https://github.com/WordPress/gutenberg/pull/57638/files#diff-51f269c9a5ba27a60e9fa5b9ca2b38716feb71521443e44c542f2e0423ee3dec). cc @ntsekouras

Reply:
Personally I don't think it's bad to use a specific class with a field's id suffix and I wouldn't worry right now about indicating public API, since we can have breaking changes in the DataViews package.

Having said that, I agree with you that we don't need this right now. I updated the code.